src/test/ui/closures/2229_closure_analysis/migrations/insignificant_drop_attr.rs

```diff
@@ -0,0 +1,64 @@
+#![deny(disjoint_capture_drop_reorder)]
```

Comment:
add a `// run-pass`, since this shouldn't produce any lints. Probably what is affecting --bless as well